reference/sql-query-tables/prices/README.md

```diff
@@ -1,9 +1,27 @@
 ---
-description: Tables containing historical prices of tokens
+description: Price data tables available in SQL query
 ---
 
 # Prices
 
+### Aggregate price base type tables available to query
+
+| View Name                                          | Description                                      |
+| -------------------------------------------------- | ------------------------------------------------ |
+| [`spiceai.trades_detailed`](trades\_detailed.md)    | Detailed trade data                              |
+| [`spiceai.trades`](trades.md)                      | Trade data                                       |
+| [`spiceai.assets`](assets.md)                      | Asset data                                       |
+| [`spiceai.prices`](prices.md)                      | Price data                                       |
+
+The columns and their schema available for each table can be viewed with the `describe <table>` command. For example:
+
+```sql
+/* Show the columns available */
+DESCRIBE spiceai.prices;
+DESCRIBE spiceai.assets;
+```
+
+### Per-pair price base type tables available to query
```

Comment:
We should decide whether its worth continuing to publicise these `prices.[Token0]_[Token2]`  datasets.